model/src/test/assets/package-manager-repository-configuration.ort.yml

```diff
@@ -0,0 +1,7 @@
+analyzer:
+  package_managers:
+    Gradle:
+      must_run_after:
```

Comment:
Just to double-check: In my global `config.yml` I use `mustRunAfter` and it works. And that's exactly the (confusing) point, that the syntax is different between `config.yml` and `.ort.yml`, right?

Reply:
Correct.

Reply:
The tests now also cover the two cases:
`allow_dynamic_versions` in the added `analyzer-repository-configuration.ort.yml` file,
and `allowDynamicVersions` in the `reference.yml` file.